<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>768</height>
   </rect>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>Lumir</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="styleSheet">
    <string notr="true">QWidget
{
	background-color: rgb(38, 38, 38);
}</string>
   </property>
   <widget class="QPushButton" name="openButton">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="resource.qrc">
      <normaloff>:/images/open.png</normaloff>:/images/open.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="saveButton">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>20</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="resource.qrc">
      <normaloff>:/images/save.png</normaloff>:/images/save.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>43</width>
      <height>43</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="channelButton">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>160</y>
      <width>91</width>
      <height>32</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lantinghei SC</family>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton 
{
	border:2px groove rgb(221, 221, 221);
	border-radius: 10px;
	color: rgb(221, 221, 221);
}
QPushButton:hover
{
	border:2px groove rgb(221, 221, 221);
	border-radius: 10px;
	color: rgb(83, 83, 83);
	background-color: rgb(221, 221, 221);
}
QPushButton:pressed
{
	border:2px groove rgb(70, 160, 245);
	border-radius: 10px;
	color: rgb(221, 221, 221);
	background-color: rgb(70, 160, 245);
}
QPushButton:disabled
{
	border:2px groove rgb(100, 100, 100);
	border-radius: 10px;
	color: rgb(100, 100, 100);
}</string>
    </property>
    <property name="text">
     <string>三通道分离</string>
    </property>
   </widget>
   <widget class="QPushButton" name="undoButton">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>20</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="resource.qrc">
      <normaloff>:/images/undo.png</normaloff>:/images/undo.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="redoButton">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>20</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="resource.qrc">
      <normaloff>:/images/redo.png</normaloff>:/images/redo.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
   </widget>
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>160</y>
      <width>760</width>
      <height>470</height>
     </rect>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAsNeeded</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAsNeeded</enum>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>758</width>
       <height>468</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <widget class="QLabel" name="imageLabel">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>0</width>
        <height>0</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="toGrayButton">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>210</y>
      <width>91</width>
      <height>32</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lantinghei SC</family>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton 
{
	border:2px groove rgb(221, 221, 221);
	border-radius: 10px;
	color: rgb(221, 221, 221);
}
QPushButton:hover
{
	border:2px groove rgb(221, 221, 221);
	border-radius: 10px;
	color: rgb(83, 83, 83);
	background-color: rgb(221, 221, 221);
}
QPushButton:pressed
{
	border:2px groove rgb(70, 160, 245);
	border-radius: 10px;
	color: rgb(221, 221, 221);
	background-color: rgb(70, 160, 245);
}
QPushButton:disabled
{
	border:2px groove rgb(100, 100, 100);
	border-radius: 10px;
	color: rgb(100, 100, 100);
}</string>
    </property>
    <property name="text">
     <string>彩色转灰度</string>
    </property>
   </widget>
   <widget class="QPushButton" name="HSLButton">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>260</y>
      <width>91</width>
      <height>32</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lantinghei SC</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton 
{
	border:2px groove rgb(221, 221, 221);
	border-radius: 10px;
	color: rgb(221, 221, 221);
}
QPushButton:hover
{
	border:2px groove rgb(221, 221, 221);
	border-radius: 10px;
	color: rgb(83, 83, 83);
	background-color: rgb(221, 221, 221);
}
QPushButton:pressed
{
	border:2px groove rgb(70, 160, 245);
	border-radius: 10px;
	color: rgb(221, 221, 221);
	background-color: rgb(70, 160, 245);
}
QPushButton:disabled
{
	border:2px groove rgb(100, 100, 100);
	border-radius: 10px;
	color: rgb(100, 100, 100);
}</string>
    </property>
    <property name="text">
     <string>色相/饱和度/亮度</string>
    </property>
   </widget>
   <widget class="QPushButton" name="otsuButton">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>310</y>
      <width>91</width>
      <height>32</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lantinghei SC</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton 
{
	border:2px groove rgb(221, 221, 221);
	border-radius: 10px;
	color: rgb(221, 221, 221);
}
QPushButton:hover
{
	border:2px groove rgb(221, 221, 221);
	border-radius: 10px;
	color: rgb(83, 83, 83);
	background-color: rgb(221, 221, 221);
}
QPushButton:pressed
{
	border:2px groove rgb(70, 160, 245);
	border-radius: 10px;
	color: rgb(221, 221, 221);
	background-color: rgb(70, 160, 245);
}
QPushButton:disabled
{
	border:2px groove rgb(100, 100, 100);
	border-radius: 10px;
	color: rgb(100, 100, 100);
}</string>
    </property>
    <property name="text">
     <string>二值化:Otsu</string>
    </property>
   </widget>
   <widget class="QPushButton" name="thresholdButton">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>360</y>
      <width>91</width>
      <height>32</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lantinghei SC</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton 
{
	border:2px groove rgb(221, 221, 221);
	border-radius: 10px;
	color: rgb(221, 221, 221);
}
QPushButton:hover
{
	border:2px groove rgb(221, 221, 221);
	border-radius: 10px;
	color: rgb(83, 83, 83);
	background-color: rgb(221, 221, 221);
}
QPushButton:pressed
{
	border:2px groove rgb(70, 160, 245);
	border-radius: 10px;
	color: rgb(221, 221, 221);
	background-color: rgb(70, 160, 245);
}
QPushButton:disabled
{
	border:2px groove rgb(100, 100, 100);
	border-radius: 10px;
	color: rgb(100, 100, 100);
}</string>
    </property>
    <property name="text">
     <string>二值化:双阈值</string>
    </property>
   </widget>
   <widget class="QPushButton" name="zoomRotateButton">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>410</y>
      <width>91</width>
      <height>32</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lantinghei SC</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton 
{
	border:2px groove rgb(221, 221, 221);
	border-radius: 10px;
	color: rgb(221, 221, 221);
}
QPushButton:hover
{
	border:2px groove rgb(221, 221, 221);
	border-radius: 10px;
	color: rgb(83, 83, 83);
	background-color: rgb(221, 221, 221);
}
QPushButton:pressed
{
	border:2px groove rgb(70, 160, 245);
	border-radius: 10px;
	color: rgb(221, 221, 221);
	background-color: rgb(70, 160, 245);
}
QPushButton:disabled
{
	border:2px groove rgb(100, 100, 100);
	border-radius: 10px;
	color: rgb(100, 100, 100);
}</string>
    </property>
    <property name="text">
     <string>缩放/旋转</string>
    </property>
   </widget>
   <widget class="QPushButton" name="plusMinusMulButton">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>460</y>
      <width>91</width>
      <height>32</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lantinghei SC</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton 
{
	border:2px groove rgb(221, 221, 221);
	border-radius: 10px;
	color: rgb(221, 221, 221);
}
QPushButton:hover
{
	border:2px groove rgb(221, 221, 221);
	border-radius: 10px;
	color: rgb(83, 83, 83);
	background-color: rgb(221, 221, 221);
}
QPushButton:pressed
{
	border:2px groove rgb(70, 160, 245);
	border-radius: 10px;
	color: rgb(221, 221, 221);
	background-color: rgb(70, 160, 245);
}
QPushButton:disabled
{
	border:2px groove rgb(100, 100, 100);
	border-radius: 10px;
	color: rgb(100, 100, 100);
}</string>
    </property>
    <property name="text">
     <string>加/减/乘</string>
    </property>
   </widget>
   <widget class="QPushButton" name="contrastButton">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>210</y>
      <width>91</width>
      <height>32</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lantinghei SC</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton 
{
	border:2px groove rgb(221, 221, 221);
	border-radius: 10px;
	color: rgb(221, 221, 221);
}
QPushButton:hover
{
	border:2px groove rgb(221, 221, 221);
	border-radius: 10px;
	color: rgb(83, 83, 83);
	background-color: rgb(221, 221, 221);
}
QPushButton:pressed
{
	border:2px groove rgb(70, 160, 245);
	border-radius: 10px;
	color: rgb(221, 221, 221);
	background-color: rgb(70, 160, 245);
}
QPushButton:disabled
{
	border:2px groove rgb(100, 100, 100);
	border-radius: 10px;
	color: rgb(100, 100, 100);
}</string>
    </property>
    <property name="text">
     <string>对比度调节</string>
    </property>
   </widget>
   <widget class="QPushButton" name="filterButton">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>260</y>
      <width>91</width>
      <height>32</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lantinghei SC</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton 
{
	border:2px groove rgb(221, 221, 221);
	border-radius: 10px;
	color: rgb(221, 221, 221);
}
QPushButton:hover
{
	border:2px groove rgb(221, 221, 221);
	border-radius: 10px;
	color: rgb(83, 83, 83);
	background-color: rgb(221, 221, 221);
}
QPushButton:pressed
{
	border:2px groove rgb(70, 160, 245);
	border-radius: 10px;
	color: rgb(221, 221, 221);
	background-color: rgb(70, 160, 245);
}
QPushButton:disabled
{
	border:2px groove rgb(100, 100, 100);
	border-radius: 10px;
	color: rgb(100, 100, 100);
}</string>
    </property>
    <property name="text">
     <string>平滑滤波器</string>
    </property>
   </widget>
   <widget class="QPushButton" name="edgeButton">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>310</y>
      <width>91</width>
      <height>32</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lantinghei SC</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton 
{
	border:2px groove rgb(221, 221, 221);
	border-radius: 10px;
	color: rgb(221, 221, 221);
}
QPushButton:hover
{
	border:2px groove rgb(221, 221, 221);
	border-radius: 10px;
	color: rgb(83, 83, 83);
	background-color: rgb(221, 221, 221);
}
QPushButton:pressed
{
	border:2px groove rgb(70, 160, 245);
	border-radius: 10px;
	color: rgb(221, 221, 221);
	background-color: rgb(70, 160, 245);
}
QPushButton:disabled
{
	border:2px groove rgb(100, 100, 100);
	border-radius: 10px;
	color: rgb(100, 100, 100);
}</string>
    </property>
    <property name="text">
     <string>边缘检测</string>
    </property>
   </widget>
   <widget class="QPushButton" name="houghButton">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>360</y>
      <width>91</width>
      <height>32</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lantinghei SC</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton 
{
	border:2px groove rgb(221, 221, 221);
	border-radius: 10px;
	color: rgb(221, 221, 221);
}
QPushButton:hover
{
	border:2px groove rgb(221, 221, 221);
	border-radius: 10px;
	color: rgb(83, 83, 83);
	background-color: rgb(221, 221, 221);
}
QPushButton:pressed
{
	border:2px groove rgb(70, 160, 245);
	border-radius: 10px;
	color: rgb(221, 221, 221);
	background-color: rgb(70, 160, 245);
}
QPushButton:disabled
{
	border:2px groove rgb(100, 100, 100);
	border-radius: 10px;
	color: rgb(100, 100, 100);
}</string>
    </property>
    <property name="text">
     <string>霍夫变换</string>
    </property>
   </widget>
   <widget class="QPushButton" name="binarymorButton">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>410</y>
      <width>91</width>
      <height>32</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lantinghei SC</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton 
{
	border:2px groove rgb(221, 221, 221);
	border-radius: 10px;
	color: rgb(221, 221, 221);
}
QPushButton:hover
{
	border:2px groove rgb(221, 221, 221);
	border-radius: 10px;
	color: rgb(83, 83, 83);
	background-color: rgb(221, 221, 221);
}
QPushButton:pressed
{
	border:2px groove rgb(70, 160, 245);
	border-radius: 10px;
	color: rgb(221, 221, 221);
	background-color: rgb(70, 160, 245);
}
QPushButton:disabled
{
	border:2px groove rgb(100, 100, 100);
	border-radius: 10px;
	color: rgb(100, 100, 100);
}</string>
    </property>
    <property name="text">
     <string>二值形态学</string>
    </property>
   </widget>
   <widget class="QPushButton" name="graymorButton">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>460</y>
      <width>91</width>
      <height>32</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lantinghei SC</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton 
{
	border:2px groove rgb(221, 221, 221);
	border-radius: 10px;
	color: rgb(221, 221, 221);
}
QPushButton:hover
{
	border:2px groove rgb(221, 221, 221);
	border-radius: 10px;
	color: rgb(83, 83, 83);
	background-color: rgb(221, 221, 221);
}
QPushButton:pressed
{
	border:2px groove rgb(70, 160, 245);
	border-radius: 10px;
	color: rgb(221, 221, 221);
	background-color: rgb(70, 160, 245);
}
QPushButton:disabled
{
	border:2px groove rgb(100, 100, 100);
	border-radius: 10px;
	color: rgb(100, 100, 100);
}</string>
    </property>
    <property name="text">
     <string>灰度形态学</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cutButton">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>160</y>
      <width>91</width>
      <height>32</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lantinghei SC</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton 
{
	border:2px groove rgb(221, 221, 221);
	border-radius: 10px;
	color: rgb(221, 221, 221);
}
QPushButton:hover
{
	border:2px groove rgb(221, 221, 221);
	border-radius: 10px;
	color: rgb(83, 83, 83);
	background-color: rgb(221, 221, 221);
}
QPushButton:pressed
{
	border:2px groove rgb(70, 160, 245);
	border-radius: 10px;
	color: rgb(221, 221, 221);
	background-color: rgb(70, 160, 245);
}
QPushButton:disabled
{
	border:2px groove rgb(100, 100, 100);
	border-radius: 10px;
	color: rgb(100, 100, 100);
}</string>
    </property>
    <property name="text">
     <string>剪裁图片</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="styleSheet">
    <string notr="true">QStatusBar
{
	background-color: rgb(65, 65, 65);
}</string>
   </property>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <action name="action">
   <property name="text">
    <string>文件</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
